seldump/dbreader.py

```diff
@@ -124,8 +124,9 @@ def _fetch_objects(self):
     ) as extcondition
 from pg_class r
 join pg_namespace s on s.oid = r.relnamespace
-left join pg_depend d on d.objid = r.oid and d.deptype = 'e'
-left join pg_extension e on d.refobjid = e.oid
+left join
+    (select oid, unnest(extconfig) from pg_extension) exttable (extoid, configoid) on exttable.configoid = r.oid
+left join pg_extension e on exttable.extoid = e.oid
```

Comment:
This can be probably expressed without unnesting the array by joining if the item is in the array.

```
piro=# select 10 = any ('{10,20,30}'::int[]);
 ?column? 
----------
 t
(1 row)
```

Could you try if the following works too?

```suggestion
left join pg_extension as ec on r.oid = any(ec.extconfig) 
left join pg_extension e on ec.oid = e.oid
```

Reply:
This does indeed work, with specifying the specific pg_extension instance in the select. Patch changed.